.gitignore

```diff
@@ -12,6 +12,7 @@ result
 result-*
 .envrc
 .direnv
+locals.dhall
```

Comment:
This is not a part of our setup, please remove

Reply:
copied.